.github/workflows/android-release.yml

```diff
@@ -0,0 +1,83 @@
+#This is a workflow that creates, downloads, uploads and releases APK-file.
+name: APK Workflow
+
+#Event triggers
+#execute the jobs everytime when a pull_request to the main-branch is created
+on:
+  push:
+    branches: main
+
+jobs:
+  runtests:
+
+    name: Run Setup Tests
+    runs-on: ubuntu-latest
+    steps:
+
+      #Checkout a Git repository and set up Java JDK
+      - name: Checkout
+        uses: actions/checkout@v2.3.4
+      - name: Setup Java JDK
+        uses: actions/setup-java@v1.4.3
+        with:
+          java-version: 1.8
+
+      #command: run unit tests
+      - name: Run Build Test
+        #Working directory is the folder with the project
+        working-directory: ./BibBuddy
+        run: bash ./gradlew test --stacktrace
```

Comment:
We already run the unit tests in the android.yml workflow when we create a PR to dev. So I don't think it is necessary to run the unit tests again.

Reply:
Are there unit tests that are pushed to dev, before our workflow integration? The workflow checks the unit tests from the other branches when a pull request to dev is made. So, our workflow now does not check the unit tests that are merged to dev before our CI workflow integration.

Reply:
There were example unit tests before our workflow integration.